api-reference/40 Common Types/15 grids/ColumnAIOptions/mode.md

```diff
@@ -5,7 +5,14 @@ default: 'auto'
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Configures whether the column sends AI requests automatically or manually.
```

Comment:
A column can send a request automatically. But manual mode implies that something else initiates those requests. So you can't say "column sends a request manually".

```suggestion
Configures whether the column sends AI requests automatically or on demand.
```